File: 2015/src/main/java/io/frutsel_/aoc/Aoc.java
package io.frutsel_.aoc;

import io.frutsel_.aoc.common.AocBase;

public class Aoc extends AocBase {

    public Aoc() {
        super("2015");
    }

    public static void main(String[] args) {
        var aoc = new Aoc();

        aoc.init(args);
        aoc.runDays();
        aoc.report();
    }
}

File: 2020/kotlin/src/main/kotlin/io/frutsel_/aoc/Aoc.kt
package io.frutsel_.aoc

import io.frutsel_.aoc.common.AocBase

class Aoc : AocBase("2020")

fun main(args: Array<String>) {

    val aoc = Aoc()
    aoc.init(args)

    val output = StringBuilder("Advent of Code 2020").appendLine()

    aoc.findDays().forEach { day ->
        output.appendLine()
            .appendLine("Day ${day.number()}")
            .appendLine("- Part 1")
            .appendLine("  Answer: ${day.part1()}")
            .appendLine("- Part 2")
            .appendLine("  Answer: ${day.part2()}")
    }

    print(output)
}
